DisCatSharp/Clients/DiscordClient.Events.cs

```diff
@@ -260,6 +260,18 @@ public sealed partial class DiscordClient
 		remove => this._guildIntegrationsUpdated.Unregister(value);
 	}
 	private AsyncEvent<DiscordClient, GuildIntegrationsUpdateEventArgs> _guildIntegrationsUpdated;
+
+	/// <summary>
+	/// Fired when a guild audit log entry was created.
+	/// </summary>
+	public event AsyncEventHandler<DiscordClient, AsyncEventArgs> GuildAuditLogEntryCreated
```

Comment:
```suggestion
	public event AsyncEventHandler<DiscordClient, GuildAuditLogEntryCreateEventArgs> GuildAuditLogEntryCreated
```

TODO: Create object for event